AsyncDisplayKit/TextKit/ASTextKitRenderer+TextChecking.mm

```diff
@@ -75,6 +74,7 @@ - (NSTextCheckingResult *)textCheckingResultAtPoint:(CGPoint)point
     truncationTokenRange = { 0, truncationAttributedString.length };
   }
 
+  NSRange visibleRange = self.truncater.firstVisibleRange;
   truncationTokenRange.location += NSMaxRange(visibleRange);
```

Comment:
What does this actually do with NSNotFound, 0 ?  I like the general intent with using NSNotFound, but I'm thinking we should actually use 0, 0 as the default value because it is more likely to behave as a "no-op" in various calculations.